src/simtools/schemas/model_parameter_and_data_schema.metaschema.yml

```diff
@@ -362,8 +362,10 @@ definitions:
       - double
```

Comment:
Should we then remove the non numpy dtypes here?

Reply:
I kept this for now, as our current database is using doubles. We don't have a rule, but a little bit of backwards compatibility is probably required.